pom.xml

```diff
@@ -111,6 +111,7 @@
         <version.joda-time>2.7</version.joda-time>
         <version.jsoup>1.8.1</version.jsoup>
         <version.net.jcip>1.0</version.net.jcip>
+        <version.org.antlr-34>3.4</version.org.antlr-34>
```

Comment:
No better names come to mind.  When we upgrade from ANTLR 3.4 to a later major/minor, we can change the name also to reflect that.